src/Hamiltonians/HubbardRealSpace.jl

```diff
@@ -141,24 +283,25 @@ end
 
 # struct ================================================================================ #
 """
-    HubbardRealSpace(address; geometry=PeriodicBoundaries(M,), t=ones(C), u=ones(C, C), v=zeros(C, D))
+    HubbardRealSpace(address; geometry=PeriodicBoundaries(M,), t=1, u=0, w=0, v=0) <:
+        AbstractHamiltonian
 
 Hubbard model in real space. Supports single or multi-component Fock state
 addresses (with `C` components) and various (rectangular) lattice geometries
 in `D` dimensions.
 
 ```math
-  \\hat{H} = -\\sum_{\\langle i,j\\rangle,σ} t_σ a^†_{iσ} a_{jσ} +
-  \\frac{1}{2}\\sum_{i,σ} u_{σσ} n_{iσ} (n_{iσ} - 1) +
-  \\sum_{i,σ≠τ}u_{στ} n_{iσ} n_{iτ}
+  Ĥ = -∑_{⟨i,j⟩,σ} t_{i,σ} a^†_{i,σ} a_{j,σ} +
+  \\frac{1}{2} ∑_{i,σ,σ'} u_{σ,σ'} n_{i,σ} (n_{i,σ'} - 1) +
+  ∑_{⟨i,j⟩,σ,σ'} w_{σ,σ'} n_{i,σ} n_{j,σ'}
```

Comment:
The contributions to the on-site interactions from different components are not handled correctly. 
Please double check that the code implementation is correct!

```suggestion
  Ĥ = -∑_{⟨i,j⟩,σ} t_{i,σ} a^†_{i,σ} a_{j,σ} +
   ½ ∑_{i,σ,σ'} u_{σ,σ'} n_{i,σ} (n_{i,σ'} - δ_{σ,σ'}) +
  ∑_{⟨i,j⟩,σ,σ'} w_{σ,σ'} n_{i,σ} n_{j,σ'}
```